src/commands/publish/mod.rs

```diff
@@ -9,10 +9,11 @@ pub use package::Package;
 use crate::settings::target::KvNamespace;
 use route::Route;
 
-use upload_form::build_script_and_upload_form;
+use upload_form::build_upload_form;
```

Comment:
this stutter is a bummer.

Reply:
hardly the end of the world though!

Reply:
yah but the fix was easy so i added another commit